config/doxx.coffee

```diff
@@ -13,7 +13,8 @@ module.exports = {
   templatesDir: config.templatesDir
   partialsDir: config.partialsDir
   metaExtra: (fileName) ->
-    improveDocsLink: "#{config.editPageLink}/#{config.docsSourceDir}/#{fileName}"
+    [..., newName] = fileName.split "/"
```

Comment:
What does `new` mean in this context? `lastPathSegment` might be clearer (or something like that)